### PR TITLE
Do not upload _rawData for mapped relationship target entities

### PR DESCRIPTION
**Pull request by austinkelleher**

### CODE REVIEW

File: src/steps/resource-manager/converters.ts [hunk 1]
```diff
@@ -119,7 +119,7 @@ export function createOrganizationEntity(
         _type: ORGANIZATION_ENTITY_TYPE,
         _class: ORGANIZATION_ENTITY_CLASS,
         name: data.name,
-        displayName: data.name as string,
+        displayName: data.displayName as string,
```
Comment (austinkelleher) [inline src/steps/resource-manager/converters.ts:122]: Bonus change

File: src/steps/resource-manager/index.ts [hunk 2]
```diff
@@ -308,7 +308,10 @@ export async function buildOrgFolderProjectMappedRelationships(
             relationshipDirection: RelationshipDirection.FORWARD,
             sourceEntityKey: organizationEntity._key,
             targetFilterKeys: [['_type', '_key']],
-            targetEntity: projectEntity,
+            targetEntity: {
+              ...projectEntity,
+              _rawData: undefined,
```
Comment (austinkelleher) [inline src/steps/resource-manager/index.ts:313]: Actual fix.

File: src/steps/resource-manager/index.ts [hunk 3]
```diff
@@ -332,7 +335,10 @@ export async function buildOrgFolderProjectMappedRelationships(
               relationshipDirection: RelationshipDirection.FORWARD,
               sourceEntityKey: folderEntity._key,
               targetFilterKeys: [['_type', '_key']],
-              targetEntity: projectEntity,
+              targetEntity: {
+                ...projectEntity,
+                _rawData: undefined,
```
Comment (austinkelleher) [inline src/steps/resource-manager/index.ts:340]: Actual fix

File: src/steps/compute/index.ts [hunk 4]
```diff
@@ -393,7 +393,6 @@ export async function fetchComputeDisks(
             data: image,
             isPublic: true,
           });
-          delete imageEntity._rawData;
```
Comment (austinkelleher) [inline src/steps/compute/index.ts:396]: Bonus cleanup

**Review by ndowmon — APPROVED** [review]

**Pull request merged**